ingestion/src/metadata/ingestion/source/database/dbt/metadata.py

```diff
@@ -116,6 +117,16 @@ def __init__(self, config: WorkflowSource, metadata: OpenMetadata):
             else "dbtTags"
         )
 
+        # Create a service map to get dbt model lineage across different databases
+        all_databases = self.metadata.es_search_from_fqn(
+            entity_type=Database,
+            fqn_search_string="*",
+        )
+        self.database_services_map = {
```

Comment:
This map might fail when you will have databases with same names under different services.

Reply:
True ! I did not know it was allowed in backend